core/src/main/java/org/apache/iceberg/rest/HTTPClient.java

```diff
@@ -448,6 +443,43 @@ static HttpRequestInterceptor loadInterceptorDynamically(
     return instance;
   }
 
+  static HttpClientConnectionManager getConnectionManager(
```

Comment:
There seems to be limited utility for this function, maybe we could keep the code inline.

Reply:
Discussed offline